module2.js

```diff
@@ -22,7 +22,18 @@
 
 function question1(personObj) {
   // TODO
-  return /* ??? */;
+  return (
+    "Hi, my name is " +
```

Comment:
Great job having a go. By the way you can use `Hi, my name is ${personObj.firstName}` or destructuring like `const { firstName } = personObj`; see https://developers.google.com/web/updates/2015/01/ES6-Template-Strings